util/params/params.go

```diff
@@ -0,0 +1,21 @@
+package params
+
+type Params map[string]string
```

Comment:
```suggestion
type Parameters map[string]string
```
Full name is more clear.